### ISSUE TITLE
"Shift+Command+/" should show macOS Help menu for VS Code

### ISSUE THREAD

**Issue by jkulak** [Microsoft/vscode#71676]
- VSCode Version: Version 1.32.3 (1.32.3)
- OS Version: 10.14.4 (18E226)

Steps to Reproduce:

1. Remove default ```editor.action.jumpToBracket``` key binding
2. Restart VS Code (just in case)
3. Press "Shift+Command+/"

I would assume, that ```Shift + Command + /``` would take me to Search field in the top bar of VS Code app in the macOS. Instead it does nothing (after removing the default binding for this combination).

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
This works fine for me and I don't think that is the default keybinding. Do you have a keybindings extension installed? Does this reproduce if you start with `--disable-extensions`?

**Comment by jkulak** [comment]
You are right, it's not a default keybinding (the default I mentioned is with backslash -- not slash -- so it doesn't apply here).

I did check it with `--disable-extensions` and see that all extensions are disabled. Still doesn't work.

I did check it on my second macOS (very similar setup) and it doesn't work here either.

I don't use Butler app nor anything like this, no extra pref panes for any tricks with keyboard, extra shortcuts etc.

What I have just noticed is when I press `Options+Command+i` (which should Toggle Developer Tools from the Help menu), only the Help text blinks and it doesn't open the Developer Tools. Clicking the item with mouse opens the Developer Tools as expected. Other shortcuts with `Options+Command` like with `f` or `↑` or `↓` work fine.

**Comment by roblourens** [comment]
Does it work in other apps like Chrome? What is your macOS version?

**Comment by jkulak** [comment]
* OS Version: 10.14.4 (18E226)

Works in Chrome, Safari, Finder, Telegram, iTerm. Doesn't work in Spotify - but no idea if it should.

I turned off all the extra apps that I am using that work in the background. Still no change. I disabled all plugins. I have no `@source:user` key bindings. Assigning `Shift+cmd+/` to another action works fine. Restoring it other action to default doesn't fix the problem.
